server/target/target.go

```diff
@@ -511,13 +508,29 @@ func fetchTargetsFromOLAPDB(ctx context.Context, env environment.Env, q *query_b
 	return resp, nil
 }
 
+func computeNextPaginationToken(commitTimestamps map[string]int64) *tppb.PaginationToken {
+	var nextPageToken *tppb.PaginationToken
+	for commit, lastCreatedAtUsec := range commitTimestamps {
+		if nextPageToken == nil {
+			nextPageToken = &tppb.PaginationToken{
+				InvocationEndTimeUsec: lastCreatedAtUsec,
+				CommitSha:             commit,
+			}
+		} else if lastCreatedAtUsec < nextPageToken.GetInvocationEndTimeUsec() || (lastCreatedAtUsec == nextPageToken.GetInvocationEndTimeUsec() && commit > nextPageToken.GetCommitSha()) {
```

Comment:
InvocationEndTimeUsec is a bit of a misnomer, right?  it's actually the start time of the last invocation the user saw?

Reply:
It's a bit strange. I don't remember why it's called invocationEndTimeUsec in the pagination token anymore.